resources/css/utilities.css

```diff
@@ -54,6 +54,10 @@
   }
 }
 
+div.phpdebugbar {
+  @apply hidden sm:block;
+}
```

Comment:
Would it be possible (would it make sense) to still show the size/branch/language?
![image](https://github.com/user-attachments/assets/840aa863-c053-401b-8604-20e0b9975943)

Reply:
Also, `master` is not correct :(
How is that determined?

Reply:
Thanks for pointing this out. I didn't even notice that all this stuff was incorrect. The locale is incorrect, too.

Unfortunately, it looks like the current branch is determined from an environment variable. After thinking things over, I've made the following changes in an attempt to make this debug element a bit more useful:

* If the environment is `local`, then the branch is no longer shown.
* Right now, language is tightly coupled to locale. I've removed display of the language.
* Locale is now correctly shown (try changing to pl_PL or pt_BR).
* On mobile, the debug content displays as a row.

![Screenshot 2024-11-09 at 3 00 50 PM](https://github.com/user-attachments/assets/518ee2e3-518b-4cf1-93b1-e3307ad465fe)

![Screenshot 2024-11-09 at 3 01 12 PM](https://github.com/user-attachments/assets/0a6b0954-fe46-461f-b4fe-ccaa418c08bd)